requirements.txt

```diff
@@ -28,7 +28,7 @@ certifi==2017.4.17        # via requests
 cffi==1.10.0              # via cryptography
 chardet==3.0.4            # via requests
 colorama==0.3.9           # via mondrian
-cryptography==2.1.4
+cryptography==2.3.1
```

Comment:
Running `pip-compile --output-file requirements.txt requirements.in` also adds `appnope==0.1.0            # via ipython` above.

Reply:
Never mind about that since it's only for ipython.